config/images/images.yaml

```diff
@@ -142,4 +142,29 @@ images:
   tags:
   - 2.8.1
   - 2.8.2
-  - 2.8.3
\ No newline at end of file
+  - 2.8.3
+# ebs-csi-driver/aws-ebs-csi-driver
+- source: public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver
```

Comment:
In https://github.com/gardener/ci-infra/pull/748#pullrequestreview-1469215924 we discussed that we only copy DockerHub images. Can you elaborate now why we need to copy GHCR and ECR images?

cc @oliver-goetz

Reply:
Additionally, the aws-ebs-csi-driver has a image in registry.k8s.io - ref https://github.com/kubernetes-sigs/aws-ebs-csi-driver#container-images. I don't really understand why we need to copy it to our GCR.

Reply:
In an IPv6 only shoot cluster you can not resolve IPv4 addresses thus it is not possible to download the images. It's required to support IPv6 in gardener.

Reply:
As shared in https://github.com/gardener/ci-infra/pull/982/files#r1379797871, please drop aws-ebs-csi-driver and use the `registry.k8s.io/provider-aws/aws-ebs-csi-driver` image in provider-aws.